scripts/src/primazactl/cmd/options/options.py

```diff
@@ -0,0 +1,98 @@
+import yaml
+from primazactl.utils import logger
+from .tenant import Tenant
+from .cluster_environment import ClusterEnvironment
+from .defaults import defaults
+
+#  apiVersion: primaza.io/v1alpha1
+#  kind: Tenant
+#  name: alice
+#  kubeconfig: ~/.kube/config
+#  context: primaza
+#  version: latest
+#  manifest_directory: ./out/config
+#  clusterEnvironments:
+#    - name: onprem-cluster
+#      environment: dev
+#      targetCluster:
+#           kubeconfig: ~/.kube/config
+#           context: onprem
+#      applicationNamespaces:
+#           - name: alice-app
+#      serviceNamespaces:
+#           - name: alice-svc
+#    - name: aws-cluster
+#      environment: dev
+#      targetCluster:
+#           kubeconfig: ~/.kube/config-aws
+#           context: aws
+#      ["applicationNamespaces:"]
+#           - name: aws-app
+#       serviceNamespaces:
+#           - name: aws-svc
+
+API_VERSION: str = "apiVersion"
+KIND: str = "kind"
+
+
+class Options(object):
+
+    options: {} = None
+    options_empty: bool = True
+
+    def __init__(self, args):
+
+        logger.log_info(f"Options:{args.options_file}:")
+
+        if args.options_file:
+            with open(str(args.options_file), "r") as options_content:
+                load_options = yaml.safe_load(options_content)
+                logger.log_info(f"loaded options: {load_options}")
+
+            if API_VERSION in load_options and \
+                    load_options[API_VERSION] == defaults["apiVersion"]:
+                if KIND in load_options and load_options[KIND] \
+                        == defaults["kind"]:
+                    self.options = load_options
+                    self.options_empty = False
+                    logger.log_info(f"Option file content: {self.options}")
+                else:
+                    logger.log_warning("Invalid \'kind\' value in options "
+                                       "file, options file will be ignored")
+            else:
+                logger.log_warning("Invalid \'apiVersion\' value in options "
+                                   "file, options file will be ignored")
```

Comment:
I made this a warning because for commands like `create tenant` the options file kind of sets the default values which may be overridden on the command line, so not always an error., but then if you specify an options file it should pass basic validity tests so will change to error.